recOrder/io/utils.py

```diff
@@ -282,91 +282,71 @@ def generic_hsv_overlay(
     return overlay_final[0] if mode == "2D" else overlay_final
 
 
-def ret_ori_overlay(
-    retardance, orientation, ret_max=10, mode="2D", cmap="JCh"
-):
+def ret_ori_overlay(retardance, orientation, ret_max=10, cmap="JCh"):
```

Comment:
This function is used in acquisition too:
https://github.com/mehta-lab/recOrder/blob/107e4c6685ed15fdab7dc4c6ffc80b5cfe901a2a/recOrder/plugin/main_widget.py#L1155-L1173

Is the plan here to stop distinguishing between 2D and (the much slower) 3D overlays?

Reply:
Then maybe remove the outdated argument from the main widget method?

Reply:
Fixed!